piptools/scripts/compile.py

```diff
@@ -245,6 +245,12 @@ def _get_default_option(option_name: str) -> Any:
     default=True,
     help="Add options to generated file",
 )
+@click.option(
+    "--unsafe-package",
+    multiple=True,
+    help="Specify a package to consider unsafe; may be used more than once. "
+    f"Replaces default unsafe packages, {', '.join(sorted(UNSAFE_PACKAGES))}",
```

Comment:
```suggestion
    f"Replaces default unsafe packages: {', '.join(sorted(UNSAFE_PACKAGES))}",
```

Shouldn't there be a colon?